wikimedia/wikipedia-ios
# Talk pages other error states

**Phabricator:** Partially https://phabricator.wikimedia.org/T312314

### Notes
* <strike>This PR was based on the other error states PR branch, PR https://github.com/wikimedia/wikipedia-ios/pull/4361 should be merged before merging this one, and the base branch should be updated to main </strike>
* This PR adds more error messages to other error states on the new Talk Pages
* It does not cover errors when creating new topics, this will be a follow-up PR

### Test Steps
1. Go to the new talk pages, try to subscribe or post a reply with no internet connection
2. Make sure an alert shows notifying the top of the screen as soon as the connection is out
3. Make sure an alert shows when trying to subscribe or post a reply
4. You can meddle with the API call when replying to force an unexpected server error to show
